request.c

```diff
@@ -1184,3 +1190,45 @@ void getClientNames(int *sock)
 	if(excludeclients != NULL)
 		clearSetupVarsArray();
 }
+
+void getUnknownQueries(int *sock)
+{
+	char server_message[SOCKETBUFFERLEN];
+
+	if(!debug)
+	{
+		sprintf(server_message,"For safety reasons, the command is only available in debugging mode!\n");
```

Comment:
Why is this?

Reply:
Initially this command was supposed to do something different, I will remove this statement here.